Rubberduck.Core/UnitTesting/IFakes.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Rubberduck.UnitTesting
```

Comment:
Nitpick / note-to-future-self: I think we should change the namespace to `Rubberduck.Core.UnitTesting` - shouldn't affect COM registration or anything.